src/styles/block-card/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  max-width: 200px;
+  border-radius: 5px;
+  border: 1px solid $background-color;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: ease 300ms;
+  }
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    padding: 32px 19px 0;
+    margin-bottom: 40px;
+
+    &-link {
+      height: 134px;
+    }
+  }
+
+  &__tittle {
+    margin-bottom: 4px;
+    padding-inline: 16px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    font-style: normal;
+    line-height: 18px;
+
+    &:hover {
+      color: #34568b;
+      transition: ease 300ms;
```

Comment:
transition should be set on default class not on :hover pseudo-class, otherwise it'll work wrong.